jdk8 build failure:java.util.function does not exist

https://ci.adoptopenjdk.net/view/Failing%20Builds/job/build-scripts/job/jobs/job/jdk8u/job/jdk8u-linux-x64-openj9/857/console
```
10:58:11  Compiling tools to generate DDR pointer classes and structure stubs
10:58:13  /home/adoptopenjdk/workspace/build-scripts/jobs/jdk8u/jdk8u-linux-x64-openj9/workspace/build/src/openj9/debugtools/DDR_VM/src/com/ibm/j9ddr/StructureReader.java:43: error: package java.util.function does not exist
10:58:13  import java.util.function.Supplier;
```
BootJDK:
```
10:25:28  Required boot JDK version: 7
10:25:28  /usr/local/gcc/bin/ccache
10:25:28  Boot jdk directory: /usr/lib/jvm/java-1.7.0-openjdk.x86_64:
10:25:28  BOOT JDK: java version "1.7.0_261"
10:25:28  BOOT JDK: OpenJDK Runtime Environment (rhel-2.6.22.1.el6_10-x86_64 u261-b02)
10:25:28  BOOT JDK: OpenJDK 64-Bit Server VM (build 24.261-b02, mixed mode)
10:25:28  JDK IN PATH: openjdk version "1.8.0_272"
10:25:28  JDK IN PATH: OpenJDK Runtime Environment (build 1.8.0_272-b10)
10:25:28  JDK IN PATH: OpenJDK 64-Bit Server VM (build 25.272-b10, mixed mode)
```

--- a/debugtools/DDR_VM/src/com/ibm/j9ddr/StructureReader.java
+++ b/debugtools/DDR_VM/src/com/ibm/j9ddr/StructureReader.java
@@ -40,7 +40,6 @@ import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
-import java.util.function.Supplier;
 import java.util.logging.Logger;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;

